proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -143,6 +143,11 @@ message QueryAllBalancesRequest {
 
   // pagination defines an optional pagination for the request.
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
+
+  // resolve_denom is the flag to resolve the denom into a human-readable form from the metadata.
+  //
+  // Since: cosmos-sdk 0.47
```

Comment:
```suggestion
  // Since: cosmos-sdk 0.48
```

Reply:
I don't think it can get in v0.47.1 as it is breaking. v0.47 has a feature freeze.